google-cloud-contrib/google-cloud-logging-adapters/cloud-logging-over-slf4j/README.md

```diff
@@ -0,0 +1,68 @@
+## Slf4j Adapter for Google Cloud Logging
+
+Built with [Slf4j](https://www.slf4j.org/) v1.7.21 and 
+[Google cloud logging library](https://github.com/GoogleCloudPlatform/google-cloud-java/tree/v0.9.2/google-cloud-logging)
+v0.9.2
```

Comment:
I would prefer not to have this version here - it is another place to keep it up to date.